src/avt/VisWindow/Colleagues/avtAnnotationWithTextColleague.C

```diff
@@ -158,23 +166,29 @@ GetStrFromTAFile(char type, int ifile, int line)
 #endif
 
     // Resolve text annotation file name
-    const std::string dotVisItFile = dotVisItDir + "/" + type + "tafile" + (char) (48+ifile) + ".txt";
-    const std::string envTmpFile = envTmpDir + "/" + type + "tafile" + (char) (48+ifile) + ".txt";
+    const std::string tstr = std::string(1,type);
+    const std::string istr = std::string(1,(char) (48+ifile));
+    const std::string theFile = "/" + tstr + "tafile" + istr + ".txt";
```

Comment:
```suggestion
    const std::string theFile = VISIT_SLASH_STRING + tstr + "tafile" + istr + ".txt";
```

Reply:
great catch

Reply:
Should have caught it with your previous merge.

Reply:
One of us should have...I had forgotten about that 😮